### PR TITLE
Clear the state when switching models.

### PR DESCRIPTION
**Pull request by huwshimi**
When switching between models we need to reset the state of the panels etc.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5160/
Test PASSed.

File: jujugui/static/gui/src/app/extensions/app-renderer-extension.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ YUI.add('app-renderer-extension', function(Y) {
         }
       }
       // If we're in sandbox we don't want to display the switcher.
-      if (this.get('sandbox')) {
+      if (this.get('sandbox') && !this.jem) {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/extensions/app-renderer-extension.js]: Drive-by to have the model switcher appear in the sandbox when there are multiple models available.

**Comment by huwshimi** [comment]
QA:
- open the profile
- click Manage on a model
- the profile should close and the model should be switched to

**Comment by huwshimi** [comment]
:shipit: to keep us moving forward.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
